<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositView</class>
 <widget class="QWidget" name="DepositView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
 background-color:  rgb(113, 137, 182);
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="keyRate">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_69">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>Tax rate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_70">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="taxRate">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_71">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_72">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>Key rate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="capitalization">
      <property name="styleSheet">
       <string notr="true">QCheckBox {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>Capitalization (monthly)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="depositTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>791</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
   background-color: white;
   color: black;
	border: 1px solid black;
	border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getDepositButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>30</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
   border-radius: 5px;
border: 1px solid white;
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
 background-color:  rgb(23, 30, 126);
border-right: 1px solid black;
border-bottom: 2px solid black;
border-right-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop: 0 rgb(195,195,195), stop: 1 rgb(240, 240, 240));
border-bottom-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop: 0 rgb(195,195,195), stop: 1 rgb(240, 240, 240));
}

QPushButton:pressed {
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.495074, fy:0.648045, stop:0.0591133 rgba(98, 118, 192, 255), stop:0.571429 rgba(222, 229, 255, 255))
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>209</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="depositSum">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="maximum">
       <double>99999999999999999209038626283633850822756121694230455365568299008.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_75">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>Deposit rate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_77">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="percentageRate">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_74">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>Deposit amount</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_73">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>months</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_76">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>Deposit term</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="fullPeriod">
      <property name="styleSheet">
       <string notr="true">QSpinBox {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_78">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: white;
	font: 16pt &quot;.AppleSystemUIFont&quot;;
}</string>
      </property>
      <property name="text">
       <string>₽</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
